Remove deprecated reference to Decentralized Identifiers (#7)
ietf-wg-scitt/draft-ietf-scitt-scrapi: draft-ietf-scitt-scrapi.md

--- abstract

This document describes a REST API that supports the normative requirements of the SCITT Architecture {{-SCITT-ARCH}}.

> Optional key discovery and query interfaces are provided to support interoperability issues with X509 Certificates and Artifact Repositories.

Reviewer: Has the code in the accompanying reference implementation from community project that supports DIDs actually been removed? It may not be required in the architecture spec (totally agree with that), but I thought SCRAPI should support it as an optional choice and this does not limit this API doesn't obsolete or prohibit that option?

Author: Hmm, we can do additional interesting things in SCRAPI, and I wouldn't limit SCRAPI identities to the one (x.509) referenced in the architecture. That said, are we planning to do anything additional with DIDs, as authors or contributors?
Thinking we can pare this down. For reference implementations, are you referring to the [scitt emulator](https://github.com/scitt-community/scitt-api-emulator) which is now out of date and we don't seem to have folks looking to update that specific codebase. Mostly due to language choice, it seems?

Reviewer: I think we are on the same page, but that means removing DIDs as one the optional identity document components for the architecture one *could* use in SCRAPI is inaccurate no?

I know what the scitt-emulator is one thing and SCRAPI should converge to it, but I would say replacing it with only x.509 is not correct.

Author: 1 is better than none, and 1 is still the first in a collection :) 
I do think it would be interesting to explore CWT as another alternative.
However, there's no recent activity in this group promoting or supporting DIDs, so cleaning it out clears the path to what we're focusing upon.

Reviewer: OK I will try to make my point more clear here with an edit. Sorry to make an editorial change on top of that that is not 100% related, but I have trouble not reading "to support interoperability issues" as problems from interoperability. We want to support improved interop, and I keep reading this to sound like we support increasing issues or problems with it. I doubt that is what is intended with this change.

```suggestion
Optional key discovery and query interfaces are provided to support interoperability between Artifact Repositories and X509 Certificates or alternatives.
```
